examples/templates/suite_com/test_com.py

```diff
@@ -55,25 +55,55 @@ def setUp(self):
     def test_run(self):
         """Thanks to the usage of dev cc_raw_loopback, let's try to send
         a message and receive it.
+        #poetry run pykiso -c examples/templates/com_aux.yaml
```

Comment:
comment inside a comment ?

Reply:
So you#re commenting for a comment inside a comment ?